lib/puppet/provider/templates/invoke_dsc_resource.ps1.erb

```diff
@@ -51,7 +51,7 @@ $invokeParams = @{
 <% end -%>
   }<% if resource.respond_to?(:dscmeta_module_version) %>
   ModuleName = @{
-    ModuleName      = <%= "\"#{resource.dscmeta_module_name}\"" %>
+    ModuleName      = <%= "\"#{vendored_modules_path}/#{resource.dscmeta_module_name}/#{resource.dscmeta_module_name}.psd1\"" %>
```

Comment:
Not sure I understand, since we check for existence of `dscmeta_module_version` (which is only generated on types that are not a part of `PSDesiredStateConfiguration` per https://github.com/puppetlabs/puppetlabs-dsc/blob/master/build/dsc/templates/dsc_type.rb.erb#L80-L82